src/tools/run-make-support/src/lib.rs

```diff
@@ -40,7 +41,13 @@ pub use python::python_command;
 pub use rustc::{aux_build, bare_rustc, rustc, Rustc};
 pub use rustdoc::{bare_rustdoc, rustdoc, Rustdoc};
 
-/// [`diff`] is implemented in terms of the [similar] library.
+/// [`ar`][mod@ar] currently uses the [ar][rust-ar] rust library, but that is subject to changes, we
```

Comment:
`mod@ar` is some `rustdoc` syntax? :eyes: That's the first time I'm seeing this :laughing:

Reply:
Yes, it's required (at the time) to distinguish `ar` the module (type namespace) versus `ar` the function (value namespace).